shared/events.js

```diff
@@ -158,6 +158,7 @@ export class HashableContract extends HashableEntry {
   // returns .data converted to vuex state using transforms and any default state
   toVuexState () {
     var state = {...this.constructor.vuex.state, ...this.data}
+    state = _.cloneDeep(state)
```

Comment:
To keep the Style Gods happy, can you keep this as one line? Just put the _.cloneDeep on line 160?